Form/Type/ChoiceFieldMaskType.php

```diff
@@ -27,16 +27,19 @@ class ChoiceFieldMaskType extends AbstractType
      */
     public function buildView(FormView $view, FormInterface $form, array $options)
     {
-        $allFieldNames = array();
+        $sanitizedMap = array();
         foreach ($options['map'] as $value => $fieldNames) {
             foreach ($fieldNames as $fieldName) {
-                $allFieldNames[$fieldName] = $fieldName;
+                $sanitizedMap[$value][] =
```

Comment:
No there is no warning http://php.net/manual/en/function.array-push.php see the second note